Lib/threading.py

```diff
@@ -895,6 +895,24 @@ class is implemented.
                 except AttributeError:
                     pass
 
+        # Only check when a program uses pyrona
+        if is_pyrona_program():
```

Comment:
I wonder if we really want to add the restriction on this level. I'm a bit worried that Pyrona unaware libraries might use threads in the background and just break.

Guess this would be a good question for Waldo, asking: "How many libraries actually spawn threads?"

Reply:
Not sure how else to do it. It seems hard to mix safe and unsafe use in the same program since we don't have a way of distinguishing between intentionally safe and intentionally unsafe uses.